contest-watcher/README.md

```diff
@@ -0,0 +1,37 @@
+# Contest-Watcher
+A one stop solution for checking and notifying you about all the upcoming contests in the next day.
+
+### What it does?
+It fetches all the upcoming contest details from the clist.by api and sends you chrome notifications when the contest starts. It fetches details of contest day by day. currently the time is in IST only.
```

Comment:
Can you elaborate more on what specific kind of contest details it gives and what is Clist.by used for, it will make it easier for the reader to understand the project better.

Reply:
ok will work on improving the readme